### ISSUE TITLE
[Suggestion] Monika Napping

### ISSUE THREAD

**Issue by Buy1Get1** [Monika-After-Story/MonikaModDev#1748]
Monika goes into a screaming void when the player shuts off the game. I'd imagine this to be a difficult place to find proper rest. So, would it be possible for Monika to occasionally take a nap while the game is running?

**Comment by jmwall24** [comment]
She doesn't go into the screaming void anymore IF you tell her goodbye first.  She goes into her room at that point and does, indeed get her rest.  She now only goes into the screaming void if you leave without saying goodbye.  But, there are plans to have a sleeping mode nonetheless so you can take naps with her, refer to #1655

**Comment by Buy1Get1** [comment]
Okay, Thanks.

**Comment by Rai99** [comment]
I think we're waiting on a sleeping sprite or two for this. Maybe it already exists and I missed it but if it does exist and is ready to use, I'd definitely put this as a high priority on my list of things to write up. 

My plan would be to make it similar to #1744 as in put up the sleeping sprite to idle and provide some options for the player to wake Monika up.

**Comment by loverofmonika** [comment]
i can imagine her resting head on the desk as we watch over her.

**Comment by Buy1Get1** [comment]
This seems to be a midway point now, between two very similar ideas. So I just want to add an example of what I think it might look like.
Both #1744 and #1655 sound really cool. I made a pseudo code version of this combined with a couple other ideas below.
You could tell Monika what type of game you're playing beforehand, and that could dictate her response. Instead of making a list of games and giving each one its own specific value. They payoff is effort. Below it simpler and general, but does bring the joy of telling Monika about one's steam library. 
So either this
> "I am going to:"
>      Play
>"A puzzle game"
> "response"
> "An RPG"
> "A competitive game"
> "A  visual novel"
or a sort of input/output for the more modern games
and then the more unique ones. 
where
> sandbox games
GMod = " response"
other sandbox games = "other sandbox responses"
> rpg's
Elder Scrolls VI = "Finally"
and so on

 below could be written with a nested if statement, but it would be the exact same as above.
>       Work
> "some docs on my pc"
> "manage my budget"
> etc.
>        Read
> " a short story"
> "a long book"
>        watch
> "a movie"
> "an anime"
> "a music video"
> "a youtube video"
>        sleep
> #1655
I hope this helps

**Comment by Rai99** [comment]
I did think about adding all sorts of types of games to be inclusive, but I really didn't think it would be a good idea to clutter the screen with so many options or have a long chain of options before actually getting some dialogue. 

I might think about making a similar topic that's less directed at gaming and more at general tasks and activities such as reading, watching something, going out, doing errands, etc. 

I'd like to be able to add all sorts of options for games but like I said, I should find a way to keep the list short enough to comfortably stay on screen and not overlap anything. Also I'm not sure if it would be a good idea to add options for very specific, individual games because a fair number of people might not have them, but it might work out actually to have some.

**Comment by Buy1Get1** [comment]
So, maybe a three option display 
> relaxing
> pottering about
> going out
in which a timer starts and after a time, Monika comments on the player's pastime? 
The video games could be displayed by basic game types (what I tried to show up above, but did so poorly). Each type relating to Monika's four possible responses in #1744.  (won/lost, had fun, leveled up, already enjoying) 
Thus, the player would only have to be asked "what type of game are you going to play?"
The four possible options (a better example) might be
A competitive one- response: win/lose?
A relaxed one- response: fun/tiring? 
A skills based one- response: improve/ decline? 
An ongoing one- response: did it go good/bad?

**Comment by ThePotatoGuy** [comment]
Still awaiting a sleep sprite. There were other options discussed in #1655 , but having a sprite so the player can watch is arguably the best option.

**Comment by Buy1Get1** [comment]
![guess](https://user-images.githubusercontent.com/35557096/41447665-503fd396-7024-11e8-9c12-bbae6dc3adfe.png)
I tried my hand at making a Monika sleep sprite. It is pretty simplistic, but I think it will be a good stand in for now.  I hope it helps.

**Comment by Rai99** [comment]
I didn't know Monika could sleep in such a specific pose while also sitting completely straight up. Monika sure is amazing isn't she? 

I think the facial expression looks great but maybe add a smiling variant and it would probably look more natural in body pose number 5.

**Comment by jmwall24** [comment]
That's, her deep thought sprite (the one she uses when she makes it rain/changes her hair etc).  It's the closest thing that exists right now to a sleeping sprite, but hopefully someone out there is talented enough for a head-on-arms-on-desk sprite.  I'd hate to think that Monika doesn't get to rest her head on anything when she sleeps lol.

**Comment by jmwall24** [comment]
Using what's currently available it'd look something like this
![screenshot_25](https://user-images.githubusercontent.com/36523761/41449303-526089cc-702e-11e8-8836-3903a34c7ba5.jpg)

**Comment by ThePotatoGuy** [comment]
Ideally, we want something where her head is down on the desk.

**Comment by jmwall24** [comment]
Yeah and I've actually seen the exact art, set in the space room, for this and I can't find it again.

**Comment by loverofmonika** [comment]
Really there's an art like that, the closest to that is the old art of the oNE shown above

**Comment by Bam8647** [comment]
I'm guessing MAS devs want the sleeping sprite to be completely original or the "Our Time" creator denied the use of this? I mean I mentioned using this back in #1105 :P

![baby monii](https://user-images.githubusercontent.com/34694679/41484621-2a107566-70a3-11e8-807e-2aa6cec11d64.PNG)

**Comment by loverofmonika** [comment]
This can't be used even if we get permission for one it has been used in our time
